src/components/banner/BannerOffer.jsx

```diff
@@ -0,0 +1,21 @@
+
+import React from 'react';
+
+function BannerOffer(){
+
+    return<>
+
```

Comment:
div is the only child here so you can remove this empty `<>` `</>`

Reply:
yes , thank you